src/test/resources/feature/SignInScenarios.feature

```diff
@@ -0,0 +1,32 @@
+@Anatolie
+Feature: Sign in and Forgot password
+
+  Scenario: Login validation with correct data
+    Given User is on "Authentication" page
+    When User introduce "test01@test.com" into "Email address" field
+    And User introduce "1@aBc" into "Password" field
+    And User clicks on "Sing in" button
+    Then User is redirected to "My account" page
+
+
+  Scenario Outline: Verify error handling (Negative scenario)
```

Comment:
Could you please rename this scenario for more informative one (specify area where you are handling error)